### PR TITLE
Test case of ApolloProperties

### PR DESCRIPTION
**Pull request by meAsky**
Fixes 4820

Changes proposed in this pull request:
- Test case of ApolloProperties

### CODE REVIEW

File: sharding-orchestration/sharding-orchestration-center/sharding-orchestration-center-apollo/src/test/java/org/apache/shardingsphere/orchestration/center/instance/ApolloPropertiesTest.java [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+    @Test
+    public void assertGetValue() {
+        Properties props = new Properties();
+        props.setProperty(ApolloPropertyKey.APP_ID.getKey(), "APOLLO_SHARDING_SPHERE1");
+        props.setProperty(ApolloPropertyKey.ENV.getKey(), "PROD");
+        props.setProperty(ApolloPropertyKey.CLUSTER_NAME.getKey(), "sit");
+        props.setProperty(ApolloPropertyKey.ADMINISTRATOR.getKey(), "guest");
+        props.setProperty(ApolloPropertyKey.TOKEN.getKey(), "a6e6bbec-1eed-4ddc-bb10-6791beb982d9");
+        props.setProperty(ApolloPropertyKey.PORTAL_URL.getKey(), "http://apollo.ctrip.sit");
```
Comment (kimmking) [inline sharding-orchestration/sharding-orchestration-center/sharding-orchestration-center-apollo/src/test/java/org/apache/shardingsphere/orchestration/center/instance/ApolloPropertiesTest.java]: "http://config-service-url"  should be better here.

File: sharding-orchestration/sharding-orchestration-center/sharding-orchestration-center-apollo/src/test/java/org/apache/shardingsphere/orchestration/center/instance/ApolloPropertiesTest.java [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+import java.util.Properties;
+import static org.hamcrest.CoreMatchers.is;
+import static org.junit.Assert.assertThat;
+
+public final class ApolloPropertiesTest {
+
+    @Test
+    public void assertGetValue() {
+        Properties props = new Properties();
+        props.setProperty(ApolloPropertyKey.APP_ID.getKey(), "APOLLO_SHARDING_SPHERE1");
```
Comment (kimmking) [inline sharding-orchestration/sharding-orchestration-center/sharding-orchestration-center-apollo/src/test/java/org/apache/shardingsphere/orchestration/center/instance/ApolloPropertiesTest.java]: "APOLLO_SHARDINGSPHERE1" is better.

Comment (meAsky) [reply sharding-orchestration/sharding-orchestration-center/sharding-orchestration-center-apollo/src/test/java/org/apache/shardingsphere/orchestration/center/instance/ApolloPropertiesTest.java]: Thanks for your advice

File: sharding-orchestration/sharding-orchestration-center/sharding-orchestration-center-apollo/src/test/java/org/apache/shardingsphere/orchestration/center/instance/ApolloPropertiesTest.java (hunk 1 above)
Comment (meAsky) [reply sharding-orchestration/sharding-orchestration-center/sharding-orchestration-center-apollo/src/test/java/org/apache/shardingsphere/orchestration/center/instance/ApolloPropertiesTest.java]: Thanks for your advice

**Pull request merged**